### DISCUSSION TITLE
[Survey] Additional QoL features for other in-game mechanics

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#19] [Ideas & Feedback]
- Lab layout overlay & current room indicator: [YouTube](https://youtu.be/TyFCSSZFBVA)
   - quick way to import today's lab layout
   - toggle-able by long-pressing TAB
   - clicks on the image will move a white rectangle underneath the cursor to keep track of your position
   - creates a separate little window with information on Izaro's phases and key drop conditions

- Syndicate board interactive overlay: [YouTube](https://youtu.be/w-MennBmFuA)
   - this would be an inclusion for a later point in time, as I would have to come up with a training system again, but this time it's more complicated since the syndicate board is more dynamic than the archnemesis UI
   - hovering over a syndicate member displays their rewards for the four different divisions
   - colored highlights represent tiering which is user-configurable

**Comment by Hillgrove** [comment]
ohh.. interactive syndicate sounds amazing :)

**Comment by dovry** [comment]
I started a POE-lab script but never finished since I do it once, maybe twice each league

https://github.com/dovry/ahk/blob/master/games/poe-lab/poe-lab.ahk

I'm looking forward to the Betrayal overlay
